src/_guides/language/language-tour.md

```diff
@@ -1723,32 +1723,37 @@ assert(foo() == null);
-|Description               | Operator                                       |
-|--------------------------|------------------------------------------------|
-| unary postfix            | <code><em>expr</em>++</code>    <code><em>expr</em>--</code>    `()`    `[]`    `?[]`    `.`    `?.`    `!` |
-| unary prefix             | <code>-<em>expr</em></code>    <code>!<em>expr</em></code>    <code>~<em>expr</em></code>    <code>++<em>expr</em></code>    <code>--<em>expr</em></code>      <code>await <em>expr</em></code>    |
-| multiplicative           | `*`    `/`    `%`    `~/`                      |
-| additive                 | `+`    `-`                                     |
-| shift                    | `<<`    `>>`    `>>>`                          |
-| bitwise AND              | `&`                                            |
-| bitwise XOR              | `^`                                            |
-| bitwise OR               | `|`                                            |
-| relational&nbsp;and&nbsp;type&nbsp;test | `>=`    `>`    `<=`    `<`    `as`    `is`    `is!` |
-| equality                 | `==`    `!=`                                   |
-| logical AND              | `&&`                                           |
-| logical OR               | `||`                                           |
-| if null                  | `??`                                           |
-| conditional              | <code><em>expr1</em> ? <em>expr2</em> : <em>expr3</em></code> |
-| cascade                  | `..` &nbsp;&nbsp; `?..`                        |
-| assignment               | `=`    `*=`    `/=`    `+=`    `-=`    `&=`    `^=`    <em>etc.</em> |
+|--------------------------+------------------------------------------------+----------------------|
+|Description               | Operator                                       | Associativity        |
+|--------------------------+------------------------------------------------+----------------------|
+| unary postfix            | <code><em>expr</em>++</code>    <code><em>expr</em>--</code>    `()`    `[]`    `?[]`    `.`    `?.`    `!` | None |
+| unary prefix             | <code>-<em>expr</em></code>    <code>!<em>expr</em></code>    <code>~<em>expr</em></code>    <code>++<em>expr</em></code>    <code>--<em>expr</em></code>      <code>await <em>expr</em></code>    | None |
+| multiplicative           | `*`    `/`    `%`    `~/`                      | Left |
+| additive                 | `+`    `-`                                     | Left |
+| shift                    | `<<`    `>>`    `>>>`                          | Left |
+| bitwise AND              | `&`                                            | Left |
+| bitwise XOR              | `^`                                            | Left |
+| bitwise OR               | `|`                                            | Left |
+| relational&nbsp;and&nbsp;type&nbsp;test | `>=`    `>`    `<=`    `<`    `as`    `is`    `is!` | None |
```

Comment:
nit: should all the columns be lined up?

Reply:
The non-breaking spaces make it a bit awkward in some editors, but I'll try my best.

Reply:
Feel free to say no here. There is a reason I asked it as a question, and marked it as a nit. +)